rllib/algorithms/cql/tests/test_cql.py

```diff
@@ -51,7 +51,7 @@ def test_cql_compilation(self):
                 # RLlib algorithm (e.g. PPO or SAC).
                 actions_in_input_normalized=False,
                 # Switch on off-policy evaluation.
-                off_policy_estimation_methods=["is"],
+                off_policy_estimation_methods={"is": {"type": "is"}},
```

Comment:
Nit: I feel like making users name all estimators serves a niche case where people actually want to use the same estimator multiple times. 

I'd like the following better and still allow an optional _name argument.
[ 
   ("is", {}}, 
   (CustomOPE, {"arg1": 123}), 
   (CustomOPE, {"arg1": 456, "_name": "the_second_one"}
]

Reply:
I'd also like to get rid of users specifying estimators by string.
I know that this look convenient for us, but it's just less explicit and leaves room for error.
We should imho, where ever we can comfortably do this, make users interact with our classes and function directly rather then describing them by strings.

Reply:
Not sure, actually. The user would have to know that there is this hidden `_name` arg (which is not a c'tor kwarg of the given class!).
I do prefer the clearer solution discussed above:

```
off_policy_estimation_methods={
  "user-provided-name-1": {
    "type": [some class or class-str],
    [c'tor kwargs],
  },
  "user-provided-name-2": {
    "type": [some class or class-str],
    [c'tor kwargs],
  },
}
```

This way, it fully matches our replay buffer configs as well as our exploration configs.